test/node/src/schema.test.ts

```diff
@@ -2446,6 +2446,39 @@ for (const dialect of DIALECTS) {
         })
       }
 
+      if (dialect !== 'sqlite') {
+        describe('parse schema name', () => {
+          beforeEach(cleanup)
+          afterEach(cleanup)
+
+          it('should parse the schema from table name', async () => {
+            await ctx.db.schema.createSchema('pets').ifNotExists().execute()
+            await ctx.db.schema.createTable('pets.name').addColumn('id', 'serial').execute();
```

Comment:
The schema and table names are kinda confusing. Why is the table named `name`? And the schema `pets` since there's already a `pets` table? Change them into something else and this is good to go.

Reply:
Oh jeesh, why have I used such a dumb schema name there 😄 Well I guess this is ok then.